include/xtensor/xscalar.hpp

```diff
@@ -409,7 +410,7 @@ namespace xt
 
     private:
 
-        storage_type* p_c;
+        container_type* p_c;
```

Comment:
did you change this back on purpose? you might want to rebase.

Reply:
I made a mistake when rebasing, I will fix that.